### ISSUE TITLE
SuSE mirrors broken? 2580 bytes missing from waterfox-classic-kpe_2020.10-0_amd64.deb

### ISSUE THREAD

**Issue by mk-pmb** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#37]
Thanks for packaging waterfox!

(Original misguided report at https://github.com/MrAlex94/Waterfox/issues/1842 )

In [xUbuntu_20.04/Packages](https://download.opensuse.org/repositories/home:/hawkeye116477:/waterfox/xUbuntu_20.04/Packages), "Package: waterfox-classic-kpe" is expected to have "Size: 40618660" bytes, but the mirrors seem to be missing 2580 bytes:

1. Lookup suitable mirrors for downloading the file: `curl --head https://download.opensuse.org/repositories/home:/hawkeye116477:/waterfox/xUbuntu_20.04/amd64/waterfox-classic-kpe_2020.10-0_amd64.deb`
1. Ask your preferred mirror about file meta data, e.g. `curl --head http://widehat.opensuse.org/repositories/home:/hawkeye116477:/waterfox/xUbuntu_20.04/amd64/waterfox-classic-kpe_2020.10-0_amd64.deb`
1. It reports "Content-Length: 40616080"

**Comment by hawkeye116477** [comment]
Simply, after some time it's automatically rebuilded (when depends are updated on repo) and that new bigger version isn't available on mirrors yet. Sometimes it take bit longer to appear on mirrors. There is also problem that release number (that is after `-` sign) isn't automatically bumped, so it may be little confusing, but this bug is already reported on https://github.com/openSUSE/open-build-service/issues/834.

**Comment by mk-pmb** [comment]
Does it mean the only solution at the moment is to remove the waterfox repo from apt sources, wait, hope and retry sometimes?

**Comment by hawkeye116477** [comment]
If current version works fine for you, then you don't need to do any step. Otherwise you can try to uninstall package, purge cached package and try again in the future.

**Comment by mk-pmb** [comment]
Yes on machines that have waterfox installed, I removed the repo from apt sources so apt can at least install and upgrade other packages. On machines that don't have waterfox yet, I don't have a good solution yet. If I had saved the old deb, I could manually install it with dpkg, but I wouldn't know how to verify the signature on that.

apt currently cannot install or re-install any version of waterfox-classic-kpe, because it would only accept the one described in the Packages file, and I can't find a download link that matches the expected file size and SHA-256.
